pages/price-feeds/schedule-price-updates.mdx

```diff
@@ -8,10 +8,11 @@ The Pyth Data Association sponsors regular on-chain updates for some price feeds
 See [Sponsored Feeds](/price-feeds/sponsored-feeds) for the current list of feeds and their update parameters.
 If you would like to see additional feeds on this list, please [contact the association via this form](https://tally.so/r/nGz2jj).
 
-There are also two different tools to schedule price updates:
+There are also three different tools to schedule price updates:
 
+- [Adrastia's Pyth Price Feed Updater](schedule-price-updates/using-adrastia) is a white-glove service that automates price updates based on time and price deviations, supporting any EVM chain.
 - [Gelato](schedule-price-updates/using-gelato) provides a turnkey automation solution for scheduled updates.
 - [Scheduler](schedule-price-updates/using-scheduler) is a service that developers can run to trigger price updates when certain time or price change conditions are met.
 
-For developers comparing these two options, Gelato is simpler, in that it does not require you to operate a service.
-However, Scheduler supports more blockchains than Gelato.
+For developers comparing these three options, Adrastia and Gelato are simpler, in that they do not require you to operate a service.
+However, Gelato is available on less chains compared to Adrastia and Scheduler.
```

Comment:
```suggestion

```

Reply:
Please remove this as we keep documentation neutral.

Reply:
> Please remove this as we keep documentation neutral.

I've removed that line. How does everything look now?